### PR TITLE
Samples module:

### LINKED ISSUE #28: Example project
- show integration with regular selenium and selenide test framework.

### PR DESCRIPTION
**Pull request by echoAlexey**
- 2 tests - google search and ace editor
- Selenium Grid Launcher for local grid

closes #28

### CODE REVIEW

File: sample/src/main/java/io/sterodium/sample/LocalSeleniumGridLauncher.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private static final String NODE_PARAMS = "-role node -nodeConfig %s";
+
+    private LocalSeleniumGridLauncher() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        launchGrid();
+    }
+
+    private static String getConfigPath(String configName) {
```
Comment (volkovs) [inline sample/src/main/java/io/sterodium/sample/LocalSeleniumGridLauncher.java:23]: Minor: private method might be in the bottom

File: sample/src/test/java/io/sterodium/sample/test/GoogleSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        searchInput.click();
+        searchInput.write(SIKULI_LOGO_SEARCH);
+
+        UiComponent searchButton = sikuliHelper.find(SEARCH_BUTTON_IMG);
+        searchButton.click();
+
+        UiComponent uiComponent = sikuliHelper.find(SIKULI_LOGO_IMG);
+
+        uiComponent.click();
+    }
```
Comment (volkovs) [inline sample/src/test/java/io/sterodium/sample/test/GoogleSearchTest.java:47]: That is cool! Can we test Component Not Found scenario as well here?

Comment (echoAlexey) [inline sample/src/test/java/io/sterodium/sample/test/GoogleSearchTest.java:47]: Yes, I can add in a next pull request.

File: sample/src/main/java/io/sterodium/sample/LocalSeleniumGridLauncher.java (hunk 1 above)
Comment (echoAlexey) [inline sample/src/main/java/io/sterodium/sample/LocalSeleniumGridLauncher.java:23]: done

**Pull request merged**